saas_apps_signup/models/res_users.py

```diff
@@ -40,7 +40,7 @@ def signup_to_try(self, values, *args, **kwargs):
         saas_template_id = values.pop("saas_template_id", "")
         max_users_limit = int(values.pop("max_users_limit", 1))
         subscription_period = values.pop("period", "")
-        operator_id = int(values.pop("operator_id"))
+        operator_id = int(values.pop("operator_id","1"))
```

Comment:
That is incorrect. operator_id should be set.

Reply:
I mean it should be set in "values".

Reply:
so it should set the default template at auth_signup.py
template = request.env.ref("saas_apps.base_template")